estate/tests/test_estate_property.py

```diff
@@ -0,0 +1,57 @@
+from odoo import fields
+from odoo.exceptions import UserError
+from odoo.tests import Form, tagged
+
+from .common import EstatePropertyCommon
+
+
+@tagged("post_install", "-at_install")
+class EstatePropertyTestCase(EstatePropertyCommon):
```

Comment:
Normally we named the class equal to the file's name.

 ```suggestion
class TestEstateProperty(EstatePropertyCommon):
```